src/tools/rbd/action/ObjectMapCheck.cc

```diff
@@ -0,0 +1,74 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
I misconstrued this note to refer to librbd/operation instead of tools/rbd/action. Sorry about that; I'll merge the files.